src/components/pages/alerts/Alert.jsx

```diff
@@ -1,63 +1,85 @@
-import { makeStyles } from '@material-ui/core';
+import {useState} from 'react';
+
+import {makeStyles} from '@material-ui/core';
 import Box from '@material-ui/core/Box';
 import Card from '@material-ui/core/Card';
 import CardContent from '@material-ui/core/CardContent';
 import IconButton from '@material-ui/core/IconButton';
 import Typography from '@material-ui/core/Typography';
 
 import BellIcon from '../../../assets/UI_component_svg/BellIcon';
+import BellIcon2 from '../../../assets/UI_component_svg/BellIcon2';
 import ExampleAlert from '../../../assets/uploaded_blueprints/alerts-floor-plan-icon.jpg';
 
 const useStyles = makeStyles({
    alertCards: {
-      marginTop: '3vh'
+      border: '2px solid #FFFFFF',
+      borderRadius: '5px',
+      width: 'auto',
+      marginTop: '3vh',
+      "&:hover": {
+         borderColor: '#A2DAD9',
+         backgroundColor: '#F4FFFF',
+      }
    },
-   alertContent: {
+   cardContent: {
       display: 'flex',
-      flex: 1,
-      flexDirection: 'column',
-      justifyContent: 'space-between',
+      alignItems: 'center',
    },
    bellIcon: {
+      marginTop: "1vh",
+      marginRight: "1vw",
       height: '30%',
       width: 'auto',
    },
-   cardContent: {
-      height: '30vh',
+   alertContent: {
       display: 'flex',
-      flexDirection: 'row',
-      alignItems: 'flex-start',
-      overflow: 'hidden',
-      padding: '3vh',
-    },
+      alignSelf: 'center',
+      flex: 1,
+      flexDirection: 'column',
+      justifyContent: 'center',
+   },
    exampleMinView: {
-      flexShrink: 0,
-      height: '100%',
+      borderRadius: '5px',
       width: 'auto',
-      marginLeft: 'auto'
+      height: 'auto',
+      maxHeight: '15vh',
+      maxWidth: 'auto',
+      border: '1px solid lightgray',
    },
 });
 
 const Alert = () => {
    const classes = useStyles();
+   let [bellIcon, setBellIcon] = useState(<BellIcon2 />);
+
    return (
       <Card className={classes.alertCards}>
-         <CardContent className={classes.cardContent}>
-            <Box marginRight="1vw">
+         <CardContent
+            onMouseOver={() => setBellIcon(<BellIcon />)}
+            onMouseOut={() => setBellIcon(<BellIcon2 />)}
+            className={classes.cardContent}
+         >
+            <Box alignSelf="flex-start">
                <IconButton className={classes.bellIcon}>
-                  <BellIcon />
+                  {bellIcon}
```

Comment:
Does the B in "bellIcon" need to be capitalized? You imported it as a capitalized B.

Reply:
bellIcon is a state that can either be BellIcon or BellIcon2. The onMouseOver in CardContent sets bellIcon to be either icons, depending on if the alert is hovered over or not.